backend/src/main/kotlin/rocks/didit/sefilm/domain/Participant.kt

```diff
@@ -0,0 +1,7 @@
+package rocks.didit.sefilm.domain
+
+open class Participant(val userID: UserID)
```

Comment:
Why `open` and not `sealed`? If `sealed` you wouldn't need an `else`-branch in your `when` [here](https://github.com/cthdidIT/itbio/pull/110/commits/4267cbcf14d37308f6c9717ffbdc81fe3cc6fc81#diff-405f24072e00719662f1c70e01198366R68)